app/views/admin/pdfs/index.html.erb

```diff
@@ -10,3 +10,26 @@
     </div>
   <% end %>
 </div>
+
+<div class="row">
+  <table class="table table-condensed">
+    <thead>
+    <tr class="warning">
+      <th data-field='title'>Title</th>
+      <th data-field='updated_at'>Last Updated</th>
+      <th data-field="Edit/Delete">Edit/Delete</th>
+    </tr>
+    </thead>
+
+    <tbody>
+    <% @pdfs.each do |pdf| %>
+      <tr id="table-row" data-title= "<%= pdf.id %>">
+        <td><%= pdf.pdf_file_file_name %></td>
+        <td><%= updated_at_time(pdf)</td>
+        <td> <%= link_to "Edit", edit_admin_pdf_path(pdf), class: "btn btn-primary" %>
+          <%= link_to "Delete", admin_pdf_path(pdf), method: :destroy, class: "btn btn-primary" %> </td>
+      </tr>
+    <% end %>
+    </tbody>
+  </table>
+</div>
```

Comment:
Do you have tests that confirm pdfs work?  When I pulled down the branch and went to /admin/pdfs, the page was broken.